src/backend/templates/partials/footer.hbs

```diff
@@ -28,11 +28,16 @@
          {{#if copyright}}
          {{#if copyright.licence}}
          <p>
-         <a href="{{{copyright.licence_url}}}"> <img src="{{{copyright.logo}}}"> </a>
-         &copy; {{copyright.year}}
+        <a href="{{{copyright.licence_url}}}"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
```

Comment:
Yes, I understood now @geekyd , the image is different for a different license. 
We can't hard code it. 

@mariobehling any suggestions on what change you wanted stating this, 

> Please use the small Creative Commons buttons instead of large buttons.

Since the type of license decides image for Creative Common.